packages/client/wallets/smart-wallet/src/SmartWalletSDK.ts

```diff
@@ -39,7 +39,7 @@ export class SmartWalletSDK {
         }
 
         const crossmintService = new CrossmintWalletService(clientApiKey);
-        const errorProcessor = new ErrorProcessor(new DatadogProvider());
+        const errorProcessor = new ErrorProcessor(scwLogger);
```

Comment:
The ErrorProcessor class logs every SDK errors, we want it always to be a datadog logger so we can see what issues devs run into most during development.

Reply:
Fixed